src/sdk/CMakeLists.txt

```diff
@@ -298,14 +298,8 @@ COMMENT "pack static cxx sdk and all depend static libs into one"
 )
```

Comment:
I used to fix Malformed archive error by rm before ar. Maybe ar -crs, remove’-T’ works?

Reply:
rm `-T` work well, but it will generate about 7G static file.

Reply:
What't the reason of malformed archive from latest main ?

Reply:
has not find the root case. it compiling fails since #2846 merged

Reply:
libopenmldbsdk.a and sdk lib in ${BIN_LIBS} have the same part, may be it's the root cause. Seems like link again.

Reply:
but it seems strange that why it can works before. #2846 does not modify the lib

Reply:
> has not found the root cause. it compiling fails since #2846 merged

👍 So it seems like no big change in cmakelists, maybe reach some resources limit?